argo/template/dm.yaml

```diff
@@ -0,0 +1,76 @@
+metadata:
+  name: tipocket-dm
+  namespace: argo
+spec:
+  templates:
+    - name: tipocket-dm
+      inputs:
+        parameters:
+          - name: ns
+            default: tipocket-dm
+          - name: hub
+            default: "docker.io"
+          - name: repository
+            default: pingcap
+          - name: image_version
+            default: nightly
+          - name: storage_class
+            default: standard
+          - name: nemesis
+            default: ""
+          - name: run_time
+            default: "24h"
+          - name: loki-addr
+            default: http://gateway.loki.svc
+          - name: loki-username
+            default: loki
+          - name: loki-password
+            default: admin
+          - name: dm-mysql-version
+            default: "5.7"
+          - name: dm-mysql-storage-size
+            default: "10Gi"
+          - name: dm-mysql-enable-binlog
+            default: true
+          - name: dm-mysql-enable-gtid
+            default: true
+          - name: dm-version
+            default: "nightly"
+          - name: dm-master-replicas
+            default: 3
+          - name: dm-worker-replicas
+            default: 3
+      outputs:
+        artifacts:
+          - name: tidb-logs
+            archiveLogs: true
+            path: /var/run/tipocket-logs
+      metadata:
+        labels:
+          ns: "{{inputs.parameters.ns}}"
+      container:
+        name: tipocket
+        image: 'pingcap/tipocket:latest'
+        imagePullPolicy: Always
+        command:
+          - sh
+          - '-c'
+          - |
+            /bin/dm-pocket \
+            -namespace={{inputs.parameters.ns}} \
+            -hub={{inputs.parameters.hub}} \
+            -repository={{inputs.parameters.repository}} \
+            -storage-class={{inputs.parameters.storage_class}} \
+            -image-version={{inputs.parameters.image_version}} \
+            -nemesis={{inputs.parameters.nemesis}} \
+            -run-time={{inputs.parameters.run_time}} \
+            -loki-addr={{inputs.parameters.loki-addr}} \
+            -loki-username={{inputs.parameters.loki-username}} \
+            -loki-password={{inputs.parameters.loki-password}} \
+            - dm.mysql.version={{inputs.parameters.dm-mysql-version}} \
```

Comment:
there is a blank.

Reply:
removed in https://github.com/pingcap/tipocket/pull/220/commits/21593e7e757ea5395d2dbd6b885b21fcba9462f6.